uberfire-structure/uberfire-structure-backend/src/main/java/org/guvnor/structure/backend/organizationalunit/config/SpaceConfigStorageImpl.java

```diff
@@ -292,7 +294,36 @@ BranchPermissions getDefaultBranchPermissions(String branchName) {
     }
 
     public SpaceInfo loadSpaceInfo() {
-        return objectStorage.read(buildSpaceConfigFilePath(SPACE_INFO));
+        try {
+            return objectStorage.read(buildSpaceConfigFilePath(SPACE_INFO));
+        } catch (RuntimeException e) {
+            if (e.getCause() instanceof MarshallingException) {
+                logger.error(e.getMessage());
+                return fixContributorClassInSpaceInfo();
+            }
+            throw new RuntimeException(e);
```

Comment:
I would prefer to pass  an exception message string while creating the runtime exception

Reply:
+1, also we are not loggin the problem unless it is a MarshallingException problem.